Kitodo/src/main/java/org/kitodo/production/forms/createprocess/CreateProcessForm.java

```diff
@@ -632,13 +635,41 @@ private void saveProcessHierarchyMetadata() {
     private void saveTempProcessMetadata(TempProcess tempProcess) {
         try (OutputStream out = ServiceManager.getFileService()
                 .write(ServiceManager.getProcessService().getMetadataFileUri(tempProcess.getProcess()))) {
-            tempProcess.getWorkpiece().setId(tempProcess.getProcess().getId().toString());
-            ServiceManager.getMetsService().save(tempProcess.getWorkpiece(), out);
+            Workpiece workpiece = tempProcess.getWorkpiece();
+            workpiece.setId(tempProcess.getProcess().getId().toString());
+            if (Objects.nonNull(rulesetManagement)) {
+                setProcessTitleMetadata(workpiece);
+            }
+            ServiceManager.getMetsService().save(workpiece, out);
         } catch (IOException e) {
             Helper.setErrorMessage(e.getLocalizedMessage(), logger, e);
         }
     }
 
+    private void setProcessTitleMetadata(Workpiece workpiece) {
+        Collection<String> keysForProcessTitle = rulesetManagement.getFunctionalKeys(FunctionalMetadata.PROCESS_TITLE);
+        if (!keysForProcessTitle.isEmpty()) {
+            String processTitle = currentProcess.getProcess().getTitle();
+            addAddableMetadataRecursive(workpiece.getLogicalStructure(), keysForProcessTitle, processTitle);
+            addAddableMetadataRecursive(workpiece.getPhysicalStructure(), keysForProcessTitle, processTitle);
+        }
+    }
```

Comment:
The `addAddableMetadataRecursive` Method is not the appropriate  method to update the configured Metadata with `use="processTitle"` because these metadata can also be required metadata wich are also  configured with `maxOccurs="1"` and `minOccurs="1"` and have to occur only one time. Therefore,  they are  not inclued in the addable metadata list.

Reply:
Good catch! I changed that.